src/conshdlr.jl

```diff
@@ -47,7 +47,7 @@ callback functions.
 From SCIP's point-of-view, this objects corresponds to the SCIP_CONSHDLRDATA,
 but its memory is managed by Julia's GC.
 
-It's recommended to store a reference to your instance of `ManagedSCIP` or
+It's recommended to store a reference to your instance of `SCIPData` or
 `SCIP.Optimizer` here, so that you can use it within your callback methods.
```

Comment:
```suggestion
It's recommended to store a reference to your instance of `SCIP.Optimizer`
here so that you can use it within your callback methods.
```